### ISSUE TITLE
0.9.0 broke positional argument passing

### ISSUE THREAD

**Issue by LinusU** [kylef/Commander#88]
I haven't had time to investigate this too much yet, but updating from 0.8.0 to 0.9.0 broke [RasterizeXCAssets](https://github.com/LinusU/RasterizeXCAssets), even though my understanding was that the only breaking change should be requiring a more recent version of Swift?

When positional arguments is missing, it correctly prints that it's missing those arguments. But when they are given it prints an "Unknown Arguments" error:

```sh
$ swift run rasterizexcassets                                                                      
[0/0] Build complete!
An error occurred: Missing value for `source`

$ swift run rasterizexcassets a
[0/0] Build complete!
An error occurred: Missing value for `output`

$ swift run rasterizexcassets a b
[0/0] Build complete!
Unknown Arguments: a b
```

Here is the code used to set up the command:

```swift
func main(source: String, output: String, size: Size) throws {
    // ...
}

command(
    Argument<String>("source", description: "Source SVG file"),
    Argument<String>("output", description: "Output path for resulting asset"),
    Option<Size>("size", default: Size(width: 64, height: 64), description: "Size (in points) of the output asset (e.g. 120x120)"),
    main
).run()
```

**Comment by kylef** [comment]
I'm not able to reproduce what you are seeing. Could you please compare against a test project. I've included a tarball in https://flr.li/F1cZ7x .

```shell
$ curl -s https://flr.li/F1cZ7x | tar xv
Package.resolved
Package.swift
Sources/
Sources/test/
Sources/test/main.swift

$ cat Sources/test/main.swift 
import Commander

func main(source: String, output: String, size: String) throws {
    // ...
    print("Source \(source)")
    print("Output \(output)")
    print("Size \(size)")
}

command(
    Argument<String>("source", description: "Source SVG file"),
    Argument<String>("output", description: "Output path for resulting asset"),
    Option<String>("size", default: "64x64", description: "Size (in points) of the output asset (e.g. 120x120)"),
    main
).run()

$ swift run test
...
Creating working copy for https://github.com/kylef/Commander
Working copy of https://github.com/kylef/Commander resolved at 0.9.2
[19/19] Build complete!
An error occurred: Missing value for `source`

$ swift run test a
[0/0] Build complete!
An error occurred: Missing value for `output`

$ swift run test a b
[0/0] Build complete!
Source a
Output b
Size 64x64

$ swift run test a b --size 128x128
[0/0] Build complete!
Source a
Output b
Size 128x128
```

**Comment by LinusU** [comment]
Sorry for not getting to this, I will investigate further to se what the problem is, will reopen if I run into more problems. Thank you for taking time on this!
